test/server/services/user.js

```diff
@@ -109,6 +109,19 @@ describe('User service tests', () => {
           done();
         }, done);
     });
+    it('should retrieve all users', (done) => {
+      User.getAllUsers()
+        .then((users) => {
+          const { models } = users;
+          const [entry1, entry2] = models;
```

Comment:
maybe don't need to change but could this have been
`const { models: [entry1, entry2] } = users;` ?